### PR TITLE
NPM install/rebuild support

### PR DESCRIPTION
**Pull request by FriggaHel**
Implement NPM modules support for playwright.

Need https://github.com/saucelabs/sauce-runner-utils/pull/5 + a new release

### CODE REVIEW

**Review by dpgraham — APPROVED** [review]

**Review by tianfeng92 — APPROVED** [review]

File: tests/run.sh [hunk 1]
```diff
@@ -14,10 +14,10 @@ for i in ${tests[@]}; do
     tmpfile=$(mktemp)
 
     echo "Running ${key}:"
-    pushd ./tests/fixtures/${key}/ > /dev/null
-    ${SAUCECTL} run -c .sauce/config.yml > ${tmpfile} 2>&1
+    pushd ./tests/fixtures/${key}/ # > /dev/null
+    ${SAUCECTL} run -c .sauce/config.yml # > ${tmpfile} 2>&1
     RETURN_CODE=${?}
-    popd > /dev/null
+    popd # > /dev/null
```
Comment (vrunoa) [inline tests/run.sh:20]: @FriggaHel was this here for debugging? can we remove the `#`?

File: tests/unit/src/folio-runner.spec.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+jest.mock('sauce-testrunner-utils');
 jest.mock('../../../src/reporter');
 const path = require('path');
 const { run } = require('../../../src/folio-runner');
 const childProcess = require('child_process');
 const { EventEmitter } = require('events');
 const SauceLabs = require('saucelabs').default;
 const fs = require('fs');
-const utils = require('../../../src/utils');
+const remoteUtils = require('sauce-testrunner-utils');
```
Comment (dpgraham) [inline tests/unit/src/folio-runner.spec.js:12]: Can we call this `testRunnerUtils`? "remote" makes me think of a remote host.

Comment (vrunoa) [inline tests/unit/src/folio-runner.spec.js:12]: what do you think about renaming this to be more "sauce-runner" specific `runnerUtils`, `sttUtils`?

File: Dockerfile [hunk 3]
```diff
@@ -100,6 +100,14 @@ ENV IMAGE_NAME=saucelabs/stt-playwright-node
 ARG BUILD_TAG
 ENV IMAGE_TAG=${BUILD_TAG}
 
+# Let saucectl know where to mount files
+RUN mkdir -p /home/seluser/__project__/ && chown seluser:seluser /home/seluser/__project__/
+LABEL com.saucelabs.project-dir=/home/seluser/__project__/
+ENV SAUCE_PROJECT_DIR=/home/seluser/__project__/
+
+# Let saucectl know what command to execute
+LABEL com.saucelabs.executable=/home/seluser/bin/folio
```
Comment (vrunoa) [inline Dockerfile:109]: @FriggaHel is this what saucectl will execute instead of `npm test`? what do you think about calling it `entrypoint`?

Comment (FriggaHel) [reply Dockerfile:109]: It will later, make I'll rename it.

**Review by dpgraham — APPROVED** [review]

**Review by vrunoa — APPROVED** [review]

**Pull request merged**